RecoPixelVertexing/PixelTrackFitting/src/PixelFitterByHelixProjections.cc

```diff
@@ -104,36 +104,20 @@ reco::Track* PixelFitterByHelixProjections::run(
   int nhits = hits.size();
   if (nhits <2) return 0;
 
-  vector<GlobalPoint> points(nhits);
-  vector<GlobalError> errors(nhits);
-  vector<bool> isBarrel(nhits);
-  
-  if (theTrackerWatcher.check(es)) {
-    edm::ESHandle<TrackerGeometry> trackerESH;
-    es.get<TrackerDigiGeometryRecord>().get(trackerESH);
-    theTracker = trackerESH.product();
-  }
-
-  if (theFieldWatcher.check(es)) {
+  if (!theField) { 
     edm::ESHandle<MagneticField> fieldESH;
     es.get<IdealMagneticFieldRecord>().get(fieldESH);
     theField = fieldESH.product();
```

Comment:
please remind me the location where it's used like this in PixelTracker.
Thanks.